.github/workflows/docker-image.yaml

```diff
@@ -73,9 +77,17 @@ jobs:
             RELEASE=${{ inputs.release }}
             COMMIT=${{ inputs.commit }}
             BUILD_DATE=${{ inputs.build_date }}
-      
+
+      - name: Install Cosign
+        if: inputs.sign
+        uses: sigstore/cosign-installer@03d0fecf172873164a163bbc64bed0f3bf114ed7 # v3.4.0
```

Comment:
you can bump to `sigstore/cosign-installer@dd6b2e2b610a11fd73dd187a43d57cc1394e35f9 # v3.0.5` and fix the version comment which is not correct